src/devtoolsPanel.ts

```diff
@@ -365,6 +365,44 @@ export class DevToolsPanel {
             `;
     }
 
+    private getCdnHtmlForWebview() {
+        let cdnBaseUrl = this.config.devtoolsBaseUri;
+        if (!cdnBaseUrl) {
+            // Not provided, calculate based on config
+            // TODO: CDP call to determine actual hash and stuff
+            cdnBaseUrl = this.config.useLocalEdgeWatch ? 'http://localhost:3000/vscode_app.html' : 'https://devtools.invalid';
+        }
+        const hostPath = vscode.Uri.file(path.join(this.extensionPath, 'out', 'host_beta', 'host.bundle.js'));
+        const hostUri = this.panel.webview.asWebviewUri(hostPath);
+
+        const stylesPath = vscode.Uri.file(path.join(this.extensionPath, 'out', 'common', 'styles.css'));
+        const stylesUri = this.panel.webview.asWebviewUri(stylesPath);
+
+        // the added fields for "Content-Security-Policy" allow resource loading for other file types
+        return `
+            <!doctype html>
+            <html>
+            <head>
+                <meta http-equiv="content-type" content="text/html; charset=utf-8">
+                <meta name="referrer" content="no-referrer">
+                <link href="${stylesUri}" rel="stylesheet"/>
+                <script src="${hostUri}"></script>
+                <meta http-equiv="Content-Security-Policy"
+                    content="default-src;
+                    img-src 'self' data: ${this.panel.webview.cspSource};
+                    style-src 'self' 'unsafe-inline' ${this.panel.webview.cspSource};
+                    script-src 'self' 'unsafe-eval' ${this.panel.webview.cspSource};
+                    frame-src 'self' ${this.panel.webview.cspSource} ${cdnBaseUrl};
+                    connect-src 'self' data: ${this.panel.webview.cspSource};
+                ">
+            </head>
+            <body>
+                <iframe id="host" frameBorder="0" src="${cdnBaseUrl}?experiments=true&edgeThemes=true"></iframe>
```

Comment:
nit: I suggest to change the id of the iframe as the real host would be the window created by the webview and could cause confusion